src/components/sbb-toggle-check/sbb-toggle-check.tsx

```diff
@@ -0,0 +1,108 @@
+import { Component, Prop, h, Event, EventEmitter } from '@stencil/core';
+import { InterfaceCheckToggleAttributes } from './sbb-link.custom';
+
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-toggle-check.scss',
+  tag: 'sbb-toggle-check',
+})
+export class SbbToggleCheck {
+  private _checkbox: HTMLInputElement;
+
+  /** Whether the toggle-check is checked. */
+  @Prop({ mutable: true }) public checked = false;
+
+  /** name of the toggle-check */
+  @Prop() public name?: string;
+
+  /** id of the toggle-check */
+  @Prop() public toggleId: string;
```

Comment:
Also initialize this with a generated id: e.g. `sbb-checkbox-${++nextId}` and declare a `let nextId = 0` above the component